src/index.ts

```diff
@@ -191,8 +189,6 @@ addHandler(
 const threadChannels = [CHANNELS.helpJs, CHANNELS.helpThreadsReact];
 addHandler(threadChannels, autothread);
 
-addHandler(CHANNELS.resumeReview, resumeReviewPdf);
```

Comment:
We are tying the image generation to the `threadCreate` and when user has not uploaded PDF in their thread and has it done in the occurring message it will not generate the image.

I think we should leave the cooldown and this implementation in-place. Perhaps there is way to differentiate when user has created thread or posted a message in already created thread.

Reply:
Gotcha, fair point on behavior change! I'll revisit

Reply:
Pushed changes!

Reply:
https://github.com/reactiflux/reactibot/assets/1551487/65078e44-ab06-4718-a233-d6d691fd6f76